[2023-07-29 03:33:48,515] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_etl_paises.crear_tabla_ manual__2023-07-29T03:33:44.992964+00:00 [queued]>
[2023-07-29 03:33:48,542] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_etl_paises.crear_tabla_ manual__2023-07-29T03:33:44.992964+00:00 [queued]>
[2023-07-29 03:33:48,544] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-29 03:33:48,546] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-07-29 03:33:48,552] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-29 03:33:48,588] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_> on 2023-07-29 03:33:44.992964+00:00
[2023-07-29 03:33:48,598] {standard_task_runner.py:52} INFO - Started process 8736 to run task
[2023-07-29 03:33:48,610] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_paises', 'crear_tabla_', 'manual__2023-07-29T03:33:44.992964+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmphrvvqutk', '--error-file', '/tmp/tmpftegp3xg']
[2023-07-29 03:33:48,614] {standard_task_runner.py:80} INFO - Job 108: Subtask crear_tabla_
[2023-07-29 03:33:48,834] {task_command.py:371} INFO - Running <TaskInstance: dag_etl_paises.crear_tabla_ manual__2023-07-29T03:33:44.992964+00:00 [running]> on host 068a3bc06d1c
[2023-07-29 03:33:48,981] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_etl_paises
AIRFLOW_CTX_TASK_ID=crear_tabla_
AIRFLOW_CTX_EXECUTION_DATE=2023-07-29T03:33:44.992964+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-29T03:33:44.992964+00:00
[2023-07-29 03:33:49,010] {base.py:68} INFO - Using connection ID 'redshift_belen' for task execution.
[2023-07-29 03:33:50,369] {dbapi.py:231} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS bapintor_coderhouse.ciudades (
    city VARCHAR PRIMARY KEY,
    "Housing" DECIMAL(10, 2),
    "Cost of Living" DECIMAL(10, 2),
    "Startups" DECIMAL(10, 2),
    "Venture Capital" DECIMAL(10, 2),
    "Travel Connectivity" DECIMAL(10, 2),
    "Commute" DECIMAL(10, 2),
    "Business Freedom" DECIMAL(10, 2),
    "Safety" DECIMAL(10, 2),
    "Healthcare" DECIMAL(10, 2),
    "Education" DECIMAL(10, 2),
    "Environmental Quality" DECIMAL(10, 2),
    "Economy" DECIMAL(10, 2),
    "Taxation" DECIMAL(10, 2),
    "Internet Access" DECIMAL(10, 2),
    "Leisure & Culture" DECIMAL(10, 2),
    "Tolerance" DECIMAL(10, 2),
    "Outdoors" DECIMAL(10, 2),
    "process_date" DATETIME DISTKEY
);, parameters: None
[2023-07-29 03:33:50,550] {postgres.py:94} INFO - INFO:  Relation "ciudades" already exists and will be skipped

[2023-07-29 03:33:50,576] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_etl_paises, task_id=crear_tabla_, execution_date=20230729T033344, start_date=20230729T033348, end_date=20230729T033350
[2023-07-29 03:33:50,664] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-29 03:33:50,747] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
